src/Mono.Android/metadata

```diff
@@ -1884,6 +1884,43 @@
+  <attr api-since="35" path="/api/package[@name='java.nio']/class[@name='MappedByteBuffer']/method[@name='compact' and count(parameter)=0]" name="managedOverride">override</attr>
+  <attr api-since="35" path="/api/package[@name='java.nio']/class[@name='MappedByteBuffer']/method[@name='compact' and count(parameter)=0]" name="compatVirtualMethod">true</attr>
+
+  <!-- Rename Namespaces -->
+  <attr api-since="35" path="/api/package[@name='android.adservices.ondevicepersonalization']" name="managedName">Android.AdServices.OnDevicePersonalization</attr>
+  <attr api-since="35" path="/api/package[@name='android.adservices.signals']" name="managedName">Android.AdServices.Signals</attr>
+  <attr api-since="35" path="/api/package[@name='java.util.random']" name="managedName">Java.Util.RandomGenerators</attr>
+
+  <!-- API-35 adds new java.util.Sequenced[Collection|Map] interfaces that existing interfaces/classes now implement. -->
+  <!-- Fix covariant return types for new 'reversed ()' method. -->	
+  <attr api-since="35" path="/api/package[@name='java.util']/interface[@name='List']/method[@name='reversed' and count(parameter)=0]" name="managedReturn">Java.Util.ISequencedCollection</attr>
+  <attr api-since="35" path="/api/package[@name='java.util']/interface[@name='Deque']/method[@name='reversed' and count(parameter)=0]" name="managedReturn">Java.Util.ISequencedCollection</attr>
+  <attr api-since="35" path="/api/package[@name='java.util']/interface[@name='NavigableSet']/method[@name='reversed' and count(parameter)=0]" name="managedReturn">Java.Util.ISequencedCollection</attr>
+  <attr api-since="35" path="/api/package[@name='java.util']/interface[@name='SortedSet']/method[@name='reversed' and count(parameter)=0]" name="managedReturn">Java.Util.ISequencedCollection</attr>
+  <attr api-since="35" path="/api/package[@name='java.util']/interface[@name='NavigableMap']/method[@name='reversed' and count(parameter)=0]" name="managedReturn">Java.Util.ISequencedMap</attr>
+  <attr api-since="35" path="/api/package[@name='java.util']/interface[@name='SortedMap']/method[@name='reversed' and count(parameter)=0]" name="managedReturn">Java.Util.ISequencedMap</attr>
+  <attr api-since="35" path="/api/package[@name='java.util']/class[@name='LinkedHashSet']/method[@name='reversed' and count(parameter)=0]" name="managedReturn">Java.Util.ISequencedCollection</attr>
+  <attr api-since="35" path="/api/package[@name='java.util']/class[@name='LinkedList']/method[@name='reversed' and count(parameter)=0]" name="managedReturn">Java.Util.ISequencedCollection</attr>
+
+  <!-- Removed because it simply narrows the method return type, which we cannot change anyways. (SequencedCollection -> SequencedSet) -->
+  <remove-node api-since="35" path="/api/package[@name='java.util']/interface[@name='SequencedSet']/method[@name='reversed' and count(parameter)=0]" />
```

Comment:
This might be another case for "reabstraction".  Though ABI-wise, i think removing this method should also work.

Reply:
In Java, this is:

```java
public interface SequencedCollection<E> extends Collection<E> {
  SequencedCollection<E> reversed();
}

public interface SequencedSet<E> extends SequencedCollection<E>, Set<E> {
  SequencedSet<E> reversed();
}
```

C# does not support covariant return type changes from implemented interfaces.  We could fix the covariant return type, but then the methods are identical, so I just removed it instead.